### PR TITLE
Page styling

### PR DESCRIPTION
**Pull request by amiraabouhadid**
Pull request for code review for project 1 of Microverse 2.0:
Positioning and Floating Elements
We were asked to make a clone of a New York Times article page using the following concepts: Embedding images The box model Flexbox Positioning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Twitter: @AbouhadidAmira
+LinkedIn: @amira-abouhadid
+
+Contributing reviews, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a star if you like this project!
+
+Acknowledgments to anyone whose code was used Inspiration etc. License This project is MIT licensed.
```
Comment (christianotieno) [inline README.md]: ## Improve `README`
- [ ] This README can be improved further to appear more stylish in a professional way. Fell free to have look at this [template ](https://github.com/microverseinc/readme-template)and draw inspiration from there. 🧑‍💼

File: index.html [hunk 2]
```diff
@@ -2,17 +2,16 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
   <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
   <link rel="stylesheet" href="./style.css">
   <meta name="viewport" content="width=device-width, initial-scale=1.0" property=''>
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
 </head>
```
Comment (christianotieno) [inline index.html:10]: ## Remove unnecessary spacing
- [ ] Kindly remove the spacing in between the heading of the main article and the main image.
- [ ] While at it. I welcome you to centre the social icons and style them to appear as they appear on the original page. See inspiration below.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)[|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|top-image|top-image-clone|

File: index.html [hunk 3]
```diff
@@ -54,58 +55,351 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
...
+        of cosmic evolution could have infused space itself with a special energy that exerted a repulsive force, causing the universe to swell faster than the speed of light for a prodigiously violent instant.</p>
+      <p>If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and irregularities. Those particles were not
+        missing, but would be diluted beyond detection, like spit in the ocean.</p>
+      <p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+      <p>On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.</p>
+      <p>Reaching back across 13.8 billion years to the first sliver of cosmic time with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of
```
Comment (christianotieno) [inline index.html:72]: ## Implement the `Editor's pick` Section
- [ ]  The `editor's pick` section is currently missing. Let's implement it, please.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|editor's-pick|editor's-pick-clone|

File: index.html [hunk 4]
```diff
@@ -54,58 +55,351 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
...
+          </div>
+          <div class="nl-1-b">
+            <b>BREAKING NEWS</b>
+          </div>
+          <div class="nl-2-b">
+            <b>Coronavirus Briefing</b>
+          </div>
+          <div class="nl-3-b">
+            <b>The Morning</b>
+          </div>
```
Comment (christianotieno) [inline index.html:274]: ## Reduce the spacing between elements
- [ ] There are far much margin and spacing in-between the elements of this section than necessary. kindly reduce the spacing and ensure the contents in these sections are centred on the page as is expected.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|suggested-section|suggested-section-clone|

File: index.html [hunk 5]
```diff
@@ -116,11 +410,118 @@ <h2>Big</h2>
...
+        <a href="">Today's Paper</a>
+        <a href="">Corrections</a>
+      </div>
+      <div>
+        <h3>OPINION</h3>
+        <a href="">Today's Opinion</a>
+        <a href="#">Op-Ed Columnists</a>
+        <a href="">Editorials</a>
+        <a href="">Op-Ed Contributors</a>
+        <a href="">Letters</a>
```
Comment (christianotieno) [inline index.html:463]: ## Improve Styling
- [ ] Improve the styling in this footer section as well. The content needs to be as centred as with the original page.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|footer|footer-clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @amiraabouhadid 👋 

A great job is done so far in this project. 👍 
I have highlighted a few issues that touch on having a much better `README`, adding some missing sections.nd improving some styling in some sections done below. Have a look 👇 

### Note:
 - Please try and stick to one `Pull Request` in-between reviews to make it easier for reviewers to track changes made in this PR. You don't need to close this PR and open a new one next time when submitting for review, thank you. 

### Happy coding 💻 
#### Christian Otieno (CR)

File: index.html (hunk 3 above)
Comment (amiraabouhadid) [reply index.html:72]: why is it showing it at the bottom of the page here then?

Screenshot from 2021-02-18 23-03-59

**Review by teekaytech — CHANGES_REQUESTED** [review]
### Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @amiraabouhadid  :wave:

__You've made laudable progress in this milestone.__
- Your linter checks all passed :100:  
- Your web page looks good :+1: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Please re-check your Readme against spelling errors (i.e. under the `Built with` section, we should have CSS3, not CC3).
- [x] The `Issues page` and `MIT license` links in your readme are broken. 
- [x] You do not have to copy the whole of your Readme to your pull request (PR) description. Your PR description should be a summary of the features you built, which you're raising the PR for.
- [x] For subsequent commits, you should improve the clarity of your commit messages. Check out [this article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) on how to write better commit messages.
- [x] Check the image below and fix the spacing problems

| Source Page | Clone Page |
| ----------------- | ---------------- |
| original | clone |
| original | clone |

- [ ] We still have some changes to make in the footer section
**Source page**
original
**Clone page**
clone



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Thank you for making those changes requested by previous TSEs. Your project has met the minimum requirements. Good work! 

Best,
Arslan Bisharat

**Pull request merged**